common/ethnicities/wc_ethnicities_orcish_rework.txt

```diff
@@ -0,0 +1,388 @@
+	}
+	gene_bs_nose_tip_angle = {
+        1 = { name = nose_tip_angle_pos    range = { 0.5 1   }    }
+	}
+	face_detail_nose_tip_def = {
+        1 = {  name = nose_tip_def             range = { 0 1 }     }
+	}
+	
+	#Ridge
+	gene_bs_nose_profile = {
+        1 = { name = nose_profile_pos    range = { 0 1 }    }
+		1 = { name = nose_profile_hawk_pos    range = { 0 0.55 }    }
+	}
+	gene_bs_nose_ridge_angle = {
+        1 = { name = nose_ridge_angle_pos    range = { 0 1 }    }
+	}
+	gene_bs_nose_ridge_width = {
+        1 = { name = nose_ridge_width_pos    range = { 0 0.8 }    }
+		1 = { name = nose_ridge_width_neg    range = { 0 0.5 }    }
+	}
+	face_detail_nose_ridge_def = {
+        1 = { name = nose_ridge_def_pos    range = { 0 1 }    }
+		1 = { name = nose_ridge_def_neg    range = { 0 1 }    }
+	}
+	
+	#Nostril
+	gene_bs_nose_nostril_height = {
+        1 = { name = nose_nostril_height_pos    range = { 0.6 1   }    }
+	}
+	gene_bs_nose_nostril_width = {
+        1 = { name = nose_nostril_width_neg    range = { 0.5 1 }    }
+	}
+	
+	###Ear
+	gene_bs_ear_lenght = {
+        1 = { name = erect_ear_lenght    range = { 0.1 0.2 } }
+    }
+	gene_bs_ear_angle = {
+        1 = { name = ear_angle_pos    range = { 0 1 } }
+	}
+	gene_bs_ear_inner_shape = {
+        1 = { name = ear_inner_shape_pos    range = { 0.18 0.18 } }
+    }
+	gene_bs_ear_bend = {
+        1 = { name = ear_upper_bend_pos    range = { 0.05 0.05 } }
+    }
+	gene_bs_ear_outward = {
+        1 = { name = ear_outward_pos    range = { 0 0.4 } }
+    }
+	gene_bs_ear_size = {
+        1 = { name = ear_size_neg    range = { 0 1 } }
+		1 = { name = ear_size_pos    range = { 0 1 } }
+    }
+	
+	###Forehead
+	gene_forehead_angle = {
+		1 = { name = forehead_angle_pos    range = { 0 0.25 }    }
+	}
+	gene_forehead_brow_height = {
+		1 = { name = forehead_brow_height_pos    range = { 0.3 0.5 }    }
+	}
+	gene_forehead_roundness = {
+		1 = { name = forehead_roundness_pos    range = { 0.5 1 }    }
+	}
+	gene_forehead_width = {
+		1 = { name = forehead_width_pos    range = { 0.3 0.7 }    }
+	}
+	gene_forehead_height = {
+		1 = { name = forehead_height_pos    range = { 0.35 0.65 }    }
+	}
+	gene_bs_forehead_brow_forward = {
+		1 = { name = forehead_brow_forward_pos    range = { 0.65 1 }    }
+	}
+	gene_bs_forehead_brow_curve = {
+		1 = { name = forehead_brow_curve_pos    range = { 0 0.5 }    }
+		1 = { name = forehead_brow_curve_neg    range = { 0 1 }    }
+	}
+	gene_bs_forehead_brow_inner_height = {
+		1 = { name = forehead_brow_inner_height_neg    range = { 0 1 }    }
+		1 = { name = forehead_brow_inner_height_pos    range = { 0 0.7 }    }
+	}
+	gene_bs_forehead_brow_outer_height = {
+		1 = { name = forehead_brow_outer_height_pos    range = { 0 1 }    }
+	}
+	gene_bs_forehead_brow_width = {
+		1 = { name = forehead_brow_width_pos    range = { 0 1 }    }
+	}
+	
+	###Head
+	gene_head_height = {
+        1 = { name = head_heightpos    range = { 0.5 0.7 }    }
```

Comment:
Typo, looks like it's what causes a crash in the Head and Neck tab of the Ruler Designer.
```suggestion
        1 = { name = head_height_pos    range = { 0.5 0.7 }    }
```